docs/developer/testing.md

```diff
@@ -0,0 +1,188 @@
+    8. Test with VoiceOver
+5. Add component to a realistic page
+    1. Import and initialize component as an outside user would
+    2. Pass typical data to the component, including handled and unhandled props
+    3. Ensure component works in typical real-world cases
+
+## Manual testing
+
+There is a [FAST-DNA documentation site for MSFT Components](https://msft-docs.fast-dna.net) that includes built in developer and testing tools to manually test.
+
+Additionally, there are services included in FAST-DNA under `./build/testing` that can be used to assist in comprehensive testing. The configuration and tests are included as a starting point and as functional and UI testing matures these may become part of the CI/CD pipeline.  As of now, these can be ran manually and the cloud services dashboard can be used to evaluate your code changes prior to submitting for code review.
+
+For those interested in using these services contact the project maintainers on discord to gain access.
+
+### On physical devices
+
+Testing local development code on physical devices in hand.
+
+Localtunnel allows you to easily share a web service on your local development machine without messing with DNS and firewall settings.
+
+Localtunnel will assign you a unique publicly accessible URL that will proxy all requests to your locally running web server.
+
+Quickstart
+
++ Install Localtunnel globally (requires NodeJS).
+
+```bash
+npm install -g localtunnel
+```
+
++ Start a web server on a local port (eg: http://localhost:8000) and use the command line interface to request a tunnel to the local server.
+
+```bash
+lt --port 8000
+```
+
++ You will receive a URL, for example https://gqgh.localtunnel.me, that you can share with anyone for as long as your local instance remains active. Any requests will be routed to your local service at the specified port.
+
+### With Sauce Labs
+
+[Sauce Labs](https://saucelabs.com/beta/dashboard/builds) allows collaborators and contributors to execute automated tests against Selenium WebDriver on pre-configured browser matrix's.
+
+The cross browser testing strategy focuses on the differences between browser rendering engines. There are three different [configuration files](https://github.com/Microsoft/fast-dna/tree/master/build/testing) based on feature maturity of ongoing work.
+
+[Alpha](https://github.com/Microsoft/fast-dna/blob/master/build/testing/config-browsers.alpha.js) Immature feature development
+
+```javascript
+node build/testing/sauce-labs/test-browsers.js alpha
+```
+
+[Beta](https://github.com/Microsoft/fast-dna/blob/master/build/testing/config-browsers.beta.js) Stable feature development
+
+```javascript
+node build/testing/sauce-labs/test-browsers.js beta
+```
+
+[Release](https://github.com/Microsoft/fast-dna/blob/master/build/testing/config-browsers.release.js) Mature feature development
+
+```javascript
+node build/testing/sauce-labs/test-browsers.js release
+```
+
+### With Applitools
```

Comment:
following the convention of your first title, this should read "Testing with Applitools"